src/ScottPlot5/ScottPlot5/Primitives/Label.cs

```diff
@@ -32,6 +32,12 @@ public class Label
     public bool AntiAliasBackground { get; set; } = true;
     public bool AntiAliasText { get; set; } = true;
 
+    /// <summary>
+    /// To make use of ClearType text, you must also set the  SKSurfaceProperties 
+    /// of your SKSurface to be something other than SKPixelGeometry.Unknown
+    /// </summary>
+    public bool SubPixelAntiAliasText { get; set; }
```

Comment:
Wonder if we should make this an enum, since setting `LcdRenderText` to true has no effect unless `paint.IsAntialias` is also true.

I'm thinking something along the lines of 
```cs
enum TextAntialiasType {
  None,
  Antialias,
  SubpixelAntialias,
}
```

Then we can set the booleans but allow a simpler API. But we'll need to keep `AntiAliasText` to keep backwards compatibility. I think we should use getters and setters to enforce the current behaviour (if `AntiAliasText` is set to true we get the current AA behaviour, if it's false we get no AA). Not sure if setting it to true while subpixel AA is on should have no effect or if it should be set to non-subpixel AA.